packages/ws-server/src/utils/utils.ts

```diff
@@ -96,7 +96,6 @@ export const sendToClient = (
   connection.send(
     JSON.stringify({
       jsonrpc: '2.0',
-      method,
```

Comment:
Looks like after removing this field, 'method' param is unused. Could you please update the param object and also the doc comment?

Reply:
Thanks for Fixing the params